src/pybind/mgr/snap_schedule/tox.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+envlist = py3
+skipsdist = true
+; toxworkdir = {env:CEPH_BUILD_DIR}/snap-schedule
+; minversion = 2.8.1
+
+[testenv]
+setenv=
+    LD_LIBRARY_PATH = {toxinidir}/../../../../build/lib
+    PATH = {toxinidir}/../../../../build/bin:$PATH
+    py27: PYTHONPATH = {toxinidir}/../../../../build/lib/cython_modules/lib.2:..
+    py3:  PYTHONPATH = {toxinidir}/../../../../build/lib/cython_modules/lib.3:{toxinidir}
+    CEPH_SNAP_SCHEDULE_UNITTEST = true
+deps =
+    pytest
+    mock
+    py27: pathlib
+commands=
+    pytest {posargs}
```

Comment:
it still would be great to have your tests part of the mgr's tox file:

* It would reduce the amount of configs (tox.ini, cmake) to maintain
* It provides more visibility to everyone working on mgr modules
* `make check` runs faster due to not having to build another tox environment
* I already removed `insights/tox`, `orchestrator/tox.ini`, `ansible/tox.ini`.